src/storage/recovery/recovery_manager.cpp

```diff
@@ -103,6 +224,14 @@ void RecoveryManager::ProcessCommittedTransaction(terrier::transaction::timestam
   txn_manager_->Commit(txn, transaction::TransactionUtil::EmptyCallback, nullptr);
 }
 
+bool RecoveryManager::IsCatalogTable(catalog::table_oid_t table_oid) {
```

Comment:
Comments for this function would really help.

Reply:
If this function is needed, it belongs under the `catalog` namespace and not under `RecoveryManager`.